src/main/java/hr/fer/dsd/privtap/rest/ActionCaller.java

```diff
@@ -0,0 +1,25 @@
+package hr.fer.dsd.privtap.rest;
+
+import hr.fer.dsd.privtap.model.action.Action;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.client.RestTemplate;
+
+import org.springframework.http.HttpHeaders;
+import java.util.Collections;
+
+public class ActionCaller {
+
+    public void callAction(String url, Action action){
+        RestTemplate template = new RestTemplate();
```

Comment:
Maybe use WebClient instead of RestTemplate because it is deprecated.
https://www.baeldung.com/spring-5-webclient